tests/test_api.rs

```diff
@@ -735,9 +735,10 @@ fn isolate_termination_methods() {
   let _setup_guard = setup();
   let mut isolate = v8::Isolate::new(Default::default());
   let handle = isolate.thread_safe_handle();
-  assert_eq!(false, isolate.terminate_execution());
-  assert_eq!(false, isolate.cancel_terminate_execution());
-  assert_eq!(false, isolate.is_execution_terminating());
+  drop(isolate);
+  assert_eq!(false, handle.terminate_execution());
+  assert_eq!(false, handle.cancel_terminate_execution());
+  assert_eq!(false, handle.is_execution_terminating());
```

Comment:
I think this is a bug fix because the test shouldn't have been passing before the upgrade, AFAICT: without the `drop(isolate)`, the isolate is alive and well, there's no reason `isolate.terminate_execution()` should return false.

The commit that added this test had a drop in it but that got removed somewhere along the way.